packages/eslint-plugin/src/rules/jsx-pragma/index.tsx

```diff
@@ -0,0 +1,147 @@
+import type { Rule } from 'eslint';
+
+import { findCompiledImportDeclarations, findDeclarationWithImport } from '../../utils/ast';
+import { addImportToDeclaration, removeImportFromDeclaration } from '../../utils/ast-string';
+
+type Options = {
+  runtime: 'classic' | 'automatic';
+};
+
+const rule: Rule.RuleModule = {
+  meta: {
+    fixable: 'code',
+    type: 'problem',
+    messages: {
+      missingPragma: 'To use CSS prop you must set the {{ pragma }} pragma.',
+      preferJsxImportSource:
+        'Use of the jsxImportSource pragma (automatic runtime) is preferred over the jsx pragma (classic runtime).',
+      preferJsx:
+        'Use of the jsx pragma (classic runtime) is preferred over the jsx pragma (automatic runtime).',
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          runtime: {
+            type: 'string',
+            pattern: '^(classic|automatic)$',
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+  },
+  create(context) {
+    const options: Options = context.options[0] || { runtime: 'automatic' };
+    const source = context.getSourceCode();
+    const jsxPragma = source.getAllComments().find((n) => n.value.indexOf('@jsx jsx') > -1);
```

Comment:
Can we store comments in a variable? I assume it's a little bit expensive

Reply:
Not sure what you mean?

Reply:
Oh I see. Yes next line. @madou use a variable for comments.